client.go

```diff
@@ -1781,6 +1810,45 @@ func (c *Client[TTx]) QueueResumeTx(ctx context.Context, tx TTx, name string, op
 	return nil
 }
 
+// queueBundle is a bundle for adding additional queues. It's made accessible
+// through Client.Queues.
+type queueBundle struct {
+	// Function that adds a producer to the associated client.
+	addProducer func(queueName string, queueConfig QueueConfig) *producer
```

Comment:
Taking a function pointer here might seem a little weird, but it lets us _not_ internalize a `client` instance, which lets us avoid putting a `TTx` generic parameter on `queueBundle`.